middlewares/auth.js

```diff
@@ -0,0 +1,16 @@
+const passport = require("passport");
+
+module.exports = (req, res, next) => {
+  passport.authenticate("jwt", { session: false }, (error, user) => {
+    if (!user || error) {
```

Comment:
Brakuje porównania tokenu z req z user.token. Teraz jak się wylogujesz to nadal na starym tokenie sprawdzisz /current